app/tests/conftest.py

```diff
@@ -17,7 +17,7 @@
 logger = logging.getLogger(__name__)
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
Before, the `app` fixture was initializing before this was called, this fixes that